mongo/pagination/pagination_test.go

```diff
@@ -15,7 +14,7 @@ const dummyCollection = "dummy_documents"
 
 type dummyDocument struct {
 	document.Base      `bson:",inline"`
-	AppID              string `bson:"app_id" json:"app_id"`
+	AppID              int    `bson:"app_id" json:"app_id"`
```

Comment:
The app ID is not an int. It's a bson ID

Reply:
In this case of tests, it just here to easily detect (with human readable field) the position of different documents. Which is import in case of testing result order.
As you can see below in the file, this field is never used for query or anything to do with logic.
The bson ID used is `_id` for the sort order test.

Reply:
The isn't it possible to name it differently? In the context of Scalingo, when I read `AppID`, I expect a bson ID. If I don't see a bson ID it triggers my attention on something that is, apparently, not important. I would prefer a different name.